scalableminds/webknossos
# Re center option

This PR adds a setting to decide whether a newly created node should be centred or not.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open a tracing
- there should be a switch setting in the annotation setting "Center new Nodes"
- when it's on new nodes should be centred; if it's off then they shouldn't
- in both cases the new node should be active
- when using ctrl+click (shortcut for not re-centring), new nodes should be marked as branchpoints, not be centred and not set active (previous behaviour)

### Issues:
- fixes #4150
------
